src/Aligent/IO/IOFileIterator.php

```diff
@@ -1,8 +1,8 @@
 <?php
+namespace Aligent\IO;
 
-require_once 'IOIteratorInterface.php';
-require_once 'IOFieldProcessor.php';
-require_once 'IO.php';
+use Aligent\IO\Exception\IOObjectAlreadyInitializedException;
```

Comment:
@kathyoungaligent i see that in here you are not calling directive use Aligent\IO\IOIteratorInterface. Is it because its not used in the class below in FileIteraror?

Reply:
IOIteratorInterface is in the same namespace as IOFileIterator, so it shouldn't need to be explicitly "used'...would it be better practice to do so?